lib/services/local/access_list.go

```diff
@@ -363,16 +385,15 @@ func (a *AccessListService) DeleteAllAccessListMembers(ctx context.Context) erro
 
 // UpsertAccessListWithMembers creates or updates an access list resource and its members.
 func (a *AccessListService) UpsertAccessListWithMembers(ctx context.Context, accessList *accesslist.AccessList, membersIn []*accesslist.AccessListMember) (*accesslist.AccessList, []*accesslist.AccessListMember, error) {
-	// Double the lock TTL to account for the time it takes to upsert the members.
-	upsertWithLockFn := func() error {
-		return a.service.RunWhileLocked(ctx, lockName(accessList.GetName()), 2*accessListLockTTL, func(ctx context.Context, _ backend.Backend) error {
-			// Create a map of the members from the request for easier lookup.
-			membersMap := make(map[string]*accesslist.AccessListMember)
 
+	if err := validateOwnerList(accessList); err != nil {
```

Comment:
@mdwn - this wasn't being done here, but I've added for parity with `UpsertAccessList()`. Is this sensible, or was it originally left out for a reason?

Reply:
I think that's sensible. The two calls were developed somewhat unrelated to one another, so this is likely just an oversight.

Reply:
I've replaced both with calls to the appropriate `CheckAndSetDefaults()`